docs/Compose.md

```diff
@@ -0,0 +1,181 @@
+# Running GUAC with Docker Compose
+
+This tutorial introduces the full GUAC ingestion pipeline and how to deploy it
+with the included Docker Compose configuration.
+
+## Demo GUAC Ingestion
+
+The GUAC demonstrations use the full GUAC GraphQL server, but use a demo-level
+all-in-one ingestion command `guacone files`. This command runs the GUAC
+collector, parser, assembler, and ingestor components all at once. These
+components are designed to be run as separate asynchronous services that combine
+to form a robust and scalable ingestion pipeline.
+
+## GUAC Components
```

Comment:
I think this section would be super helpful with that architecture diagram actually... let me send you the arch diagram, as well as some of the texts from previous docs

Reply:
We do need a full architecture doc. The main thing to focus on here, is why you want to run an ingestion pipeline vs just run `guacone`.

Reply:
done